### ISSUE TITLE
how to install and run?

### ISSUE THREAD

**Issue by sunnycqcn** [darcyabjones/panann#1]
Hello,
I think you did good job. I am very interesting to use your pipeline for predicting my fungal population. However, I can not find the manual to introduce how to install and run?
Thanks,
Fuyou

**Comment by darcyabjones** [comment]
Hi sunnycqcn,

I'm really happy that you're interested.
This pipeline is still in very early stages and isn't complete yet so it's likely to change quite a lot.
I also haven't tested it on any organisms other than my test set and my organism of interest _Parastagonospora nodorum_.

If you really want to use it soon, you'll probably have to do some coding to get it to do what you want.

I'll still try to add some documentation over the next week and get back to you.

Otherwise, you might be interested in <https://github.com/ComparativeGenomicsToolkit/Comparative-Annotation-Toolkit>.
I think I've seen some questions from you on the funannotate git repo.
Maybe using the output of funannotate as hints for augustus CPG would be a good alternative in the short term?

It depends on how many genomes you have.
For us, the time required to run and/or update Cactus genome alignments was prohibitive.

**Comment by sunnycqcn** [comment]
I am much appreciated for your replying.
I am very interesting about TE predict based on pan genome. Currently I have predicted and annotated the genes using funannotate. I have 180 isolates. I suppose TE elements should effect the virulence in these isolates. So I am interesting in about TE predict based on pan genome. If you can finish your code soon, I would like try it. Currently I the the output from funannotate.
Thanks,
Fuyou

**Comment by darcyabjones** [comment]
Hi Fuyou,

I'm sorry for the delay getting back to you.
I decided a while ago to split the gene and TE prediction into separate pipelines.

I've just finished the first releasable version of the TE prediction stuff at https://github.com/darcyabjones/pante
If you're still interested I'd love to hear how you go with it :)
Raise an issue over there if anything is unclear.

If you're also interested in the gene prediction pipeline, I'll probably have something stable in the next few weeks :)

Cheers, Darcy
